### DISCUSSION TITLE
MACE descriptors and layers

### DISCUSSION THREAD

**Discussion by zarkoivkovicc** [ACEsuit/mace#349] [Q&A] [answered]
In the documentation, it says "To get the descriptors of the first n layers, you can set the num_layers argument to n." 
Isn't that we would usually be interested only in the last n layers (the most often only the last layer)? If we denote the descriptor that we get after one layer d1 and the descriptor we get after the second layer in the model, than by default:
`get_descriptors` returns `[d2,d1]` and `get_descriptors(num_layers=1)` returns `d1`. For me it doesn't make sense, as we are mostly interested in d2. It's trivial to extract d2 from this, but am I getting something wrong here, as most of the people would be interested in d2 over d1 or [d2,d1]

**Comment by ilyes319** [comment, marked as answer]
From our experience, the first layer descriptors are smoother and more well behaved, that is why we made this decision. As you say it is trivial to extract d2.
